.github/workflows/process_opened_issues.yml

```diff
@@ -14,4 +14,4 @@ jobs:
       - uses: actions/add-to-project@7a0820f97673dfefc999713a9a6d6b7ee128bba5  # v0.0.3
         with:
           project-url: https://github.com/orgs/google/projects/20
-          github-token: ${{ secrets.WRITE_ACCESS_TOKEN }}
+          github-token: ${{ secrets.ADD_TO_PROJECT_PAT }}
```

Comment:
Can we give this a suffix "TOKEN" to match the other tokens?

Reply:
done